share/templates.d/99-generic/runtime-cleanup.tmpl

```diff
@@ -343,9 +342,6 @@ removepkg geoclue2
 removepkg cdparanoia-libs opus libtheora libvisual flac-libs gsm avahi-glib avahi-libs \
           ModemManager-glib
 
-## metacity requires libvorbis and libvorbisfile, but enc/dec are no longer needed
-removefrom libvorbis --allbut /usr/${libdir}/libvorbisfile.* /usr/${libdir}/libvorbis.*
-
```

Comment:
I am not sure about this change.
In the iso I built with metacity replaced by gnome-kiosk the libvorbis is still pulled in so removing the unneeded enc/dec is still valid and I'd keep these lines. Maybe we could update the comment with the information about what pulls the library in, but I don't insist (and I don't know).